coq-eventstruct.opam

```
@@ -18,10 +18,11 @@ operational small-step semantics for their incremental construction.
 build: [make "-j%{jobs}%" ]
 install: [make "install"]
 depends: [
-  "coq" {(>= "8.10" & < "8.14~") | (= "dev")}
+  "coq" {(>= "8.13" & <= "8.14") | (= "dev")}
```

```suggestion
  "coq" {(>= "8.13" & < "8.15~") | (= "dev")}
```
The tilda is there for a reason :) There is no `8.14` version, you want this to be patch-level unaware.

---

Unfortunately 8.14 is not supported by relation-algebra atm